install/gcp/installer/forseti_server_installer.py

```diff
@@ -78,10 +79,15 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             bool: Whether or not the deployment was successful.
             str: Deployment name.
         """
+        resources = []
```

Comment:
resources = self.composite_root_resources if self.composite_root_resources else [self.resource_root_id] ?

Reply:
This is more of a style issue. If the team and code prefers a ternary op, I'll make the change. I personally think it is easier to read, although it takes a few extra lines.